### ISSUE TITLE
Allow users to supply an array of `requests` to use when benching

### ISSUE THREAD

**Issue by thekemkid** [mcollina/autocannon#33]
Currently, you can supply the request `headers` and `body` when creating an `autocannon` instance. In future, I would like to see the ability to supply an array of objects with `headers`, `body`, `method` and `path` attributes. This might look like so:
```js
autocannon({
  url: ...,
  requests: [
    {
      method: 'POST',
      headers: {Login form data}
      path: '/api/login'
    },
    {
      method: 'GET',
      path: '/api/user/me/sensitiveData',
      headers: {auth headers}
    },
    ......
  ]
}, cb)
```

This would allow autocannon to pre-generate the http request data, and then create connections, pass in the requests, and each connection can run through the requests array repeatedly until completion.

Because of the nature of this being a load testing tool, you can not use data returned from one request in a subsequent request. This is because requests are prebuilt and cached/passed around. This would mean you must pregenerate any needed auth tokens or w/e ahead of time and embed in the testing tool.

**Comment by mcollina** [comment]
👍  get working on it.

one catch: use the `method`, `headers`, `path` and `body` specified in the top level config object as defaults.

**Comment by thekemkid** [comment]
fixed by #36
